.nvmrc

```diff
@@ -1 +1 @@
-18.13
+18.17
```

Comment:
`next` requires v18.17 or higher now. Most likely we'll bump to `20.x.x` but leaving that out of scope for this PR.